ci-config/rippled.cfg

```diff
@@ -0,0 +1,134 @@
+[server]
+port_rpc_admin_local
+port_ws_public
+port_ws_admin_local
+
+# port_peer
+# port_ws_admin_local
+# ssl_key = /etc/ssl/private/server.key
+# ssl_cert = /etc/ssl/certs/server.crt
+
+# IPs must be 0.0.0.0 instead of 127.0.0.1 to be accessed outside the docker container
+
+[port_rpc_admin_local]
+port = 5005
+ip = 0.0.0.0
+admin = 0.0.0.0
+protocol = http
+
+[port_ws_public]
+port = 80
+ip = 0.0.0.0
+protocol = ws
+
+# [port_peer]
+# port = 51235
+# ip = 0.0.0.0
+# protocol = peer
+
+[port_ws_admin_local]
+port = 6006
+ip = 0.0.0.0
+admin = 0.0.0.0
+protocol = ws
+
+[node_size]
+small
+
+# tiny
+# small
+# medium
+# large
+# huge
+
+[node_db]
+type=NuDB
+path=/var/lib/rippled/db/nudb
+advisory_delete=0
+
+# How many ledgers do we want to keep (history)?
+# Integer value that defines the number of ledgers
+# between online deletion events
+online_delete=256
+
+[ledger_history]
+# How many ledgers do we want to keep (history)?
+# Integer value (ledger count)
+# or (if you have lots of TB SSD storage): 'full'
+256
+
+[database_path]
+/var/lib/rippled/db
+
+[debug_logfile]
+/var/log/rippled/debug.log
+
+[sntp_servers]
+time.windows.com
+time.apple.com
+time.nist.gov
+pool.ntp.org
+
+[ips]
+r.ripple.com 51235
+
+[validators_file]
+validators.txt
+
+[rpc_startup]
+{ "command": "log_level", "severity": "info" }
+
+# severity (order: lots of information .. only errors)
+# debug
+# info
+# warn
+# error
+# fatal
+
+[ssl_verify]
+1
+
+# Uncomment amendments to include them as part of our CI testing
+# Note: The amendment must be included in the docker image's rippled release to be activatable.
+[features]
+Flow
```

Comment:
The `[features]` stanza no longer works. :( See https://github.com/XRPLF/xrpl-dev-portal/issues/1762#issuecomment-1441146642

Reply:
I read somewhere that it might be `amendments` now?